src/components/forms/validations.jsx

```diff
@@ -23,6 +23,7 @@ module.exports.validations = {
         return phoneNumberUtil.isValidNumber(parsed);
     }
 };
+module.exports.validations.notEqualsUsername = module.exports.validations.notEquals;
```

Comment:
yeah – this probably goes in the list of things to consider for upstream PRs to formsy we might want to do :)